style.css

```diff
@@ -61,22 +66,249 @@ p {
+  margin-bottom: 30px;
+  width: max-content;
+  align-items: center;
+  text-align: center;
+  border: #396df2 1px solid;
+}
+
+.bull:hover {
+  color: #ebebff;
+  background-color: #6070ff;
+  box-shadow: 0 8px 16pxrgba(64, 83, 252, 0.24);
+}
+
+.bull:active {
+  background: #2230d2;
+  border-radius: 8px;
+}
+
+.second h1 {
+  padding: 10px 15px;
+  font-weight: 800;
+  font-size: xx-large;
+}
+
+.lang h2 {
+  font-size: 20px;
+  line-height: 24px;
+  font-weight: 60;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.lang {
+  width: 340px;
+  padding-bottom: 40px;
+  background-color: #f7f7f9;
+}
+
+.frame {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-right: 50px;
+}
+
+.frame h2 {
+  font-size: 1.5em;
+  margin-block-start: 0.83em;
+  margin-block-end: 0.83em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  padding-bottom: 5px;
+  margin-left: 5%;
+}
+
+.pro {
+  list-style-type: none;
+}
+
+.js {
+  display: flex;
+  align-items: center;
+  margin-left: 30px;
+  padding: 5px;
+  background-color: #ebebff;
+  margin-bottom: 15px;
+  border-radius: 5px;
+}
+
+.js img {
+  padding-right: 20px;
+}
+
+.fra {
+  list-style-type: none;
+}
+
+.fro {
+  line-height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  padding: 10px 0;
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid grey;
+  font-size: 1.5em;
+  margin-block-start: 0.83em;
+  margin-block-end: 0.83em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  font-weight: bolder;
+  margin-left: 5%;
+  padding-bottom: 10px;
+  padding-right: 50px;
+}
+
+.seven {
+  background-image: url("images/Contactme.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100%;
+  color: rgb(240, 240, 240);
+}
+
+.seven h1 {
+  font-weight: 900;
+  font-size: 40px;
+  line-height: 52px;
+  font-family: 'Poppins', sans-serif;
+  padding: 10% 10%;
+  justify-content: center;
+  align-items: center;
+}
+
+.for {
+  list-style-type: none;
+}
+
+.for li {
+  padding: 3px 3px;
+  margin-left: 15px;
+}
+
+.for input {
+  width: 100%;
+  height: 15%;
+  padding: 10px 0;
+  border-radius: 8px;
+  border: rgb(240, 240, 240) 1px;
+}
+
+textarea {
+  width: 95%;
+  height: 100px;
+  padding: 20px 10px;
+  border-radius: 10px;
+}
+
+.bu {
+  margin-left: 10px;
+}
+
+.box1 {
+  justify-content: center;
+  align-items: center;
+  margin: 10% 10%;
+  padding-bottom: 10%;
+}
+
+.box1 p {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  color: #ebebff;
+}
+
+.langu {
+  background-color: #fff;
+  padding: 5px;
+}
+
+.work-about-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.work-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 30px;
+  background: rgb(245, 245, 245);
+}
+
+.work-container .second {
+  background: white;
+  margin: 15px;
+  border-radius: 15px;
+  border: 1px #dfe1e6 solid;
+}
```

Comment:
Try aligning the items inside the _second_ class with flexbox :nerd_face:

Reply:
Also, a recommendation to keep your design closer to the Figma version could be:
```
margin: 15px auto;
max-width: 90%;
```